packages/api-derive/src/util/drr.ts

```diff
@@ -12,6 +15,11 @@ import { Observable } from 'rxjs';
  */
 export const drr = () => <T>(source$: Observable<T>): Observable<T> =>
   source$.pipe(
+    catchError(err => {
+      l.error(err);
```

Comment:
... and were never logged inside the Observable loops. I added an explicit log here